pupil-spa/src/app/instructions/instructions.component.html

```diff
@@ -0,0 +1,27 @@
+<header class="page-header">
+  <h1 class="heading-xlarge">Instructions</h1>
+</header>
+
+<div class="hidden">
+  <ul class="text list list-bullet">
+    <li>This check has <strong>30 questions</strong>.</li>
```

Comment:
@jon-shipley I changed this now to pass down a count.
I prefer to add todo on ts files as a reminder.